envi/archs/arm/emu.py

```diff
@@ -1278,6 +1278,12 @@ def setThumbMode(self, thumb=1):
     def setArmMode(self, arm=1):
         self.setFlag(PSR_T_bit, not arm)
 
+    def getArmThumbMode(self):
```

Comment:
Is the plan to later convert those who just reach into getFlag() to grab this bit to use this function instead?

Reply:
possibly.  it's more addressing the fact that we have setArmMode and setThumbMode, both valuable helper functions, but we don't have any equivalent getter.